### ISSUE TITLE
Try out other small multi-lang embedding models

### ISSUE THREAD

**Issue by StephanAkkerman** [StephanAkkerman/FluentAI#54]
1. Description:

   - Problem:
     We are currently using fasttext, which is good for OOV words, but it's a bit slow and old (2015-2020). There might be better models available.

   - Solution:
     Check out the leaderboard: https://huggingface.co/spaces/mteb/leaderboard and try out models that aren't too large.

   - Prerequisites:
     [List any requirements or dependencies needed before starting.]

2. Tasks:
   - Try out newer embedding models and evaluate them.
   - Add their speed to the eval overview
   - Run the eval and save results somewhere (model name, hyperparameters etc.)
   - Remove Glove (and scipy) from eval
   - Maybe add the models to the config
   - Do we need to know what scale the model uses for similarity?
   - Clean up the models after use to free up VRAM

3. Additional context

**Comment by StephanAkkerman** [comment]
https://www.sbert.net/docs/sentence_transformer/pretrained_models.html#semantic-search

FastText has many languages support, preferably we find a model that has more than 5 languages support. We can offer some accuracy if the speed increases.

**Comment by StephanAkkerman** [comment]
https://huggingface.co/BAAI/bge-m3

**Comment by StephanAkkerman** [comment]
FastText: [Word vectors for 157 languages](https://fasttext.cc/docs/en/crawl-vectors.html)

**Comment by StephanAkkerman** [comment]
https://huggingface.co/intfloat/multilingual-e5-small: all scores are between 0.7 and 1.0 which means we need to find a method to scale it back to something more useful for us
